picard/util/__init__.py

```diff
@@ -1169,9 +1170,9 @@ def strxfrm(string):
 
 
 def detect_unicode_encoding(path):
```

Comment:
After this patch, this name doesn't look appropriate, since it detects any encoding. May be rename it to `detect_file_encoding`?

Reply:
That was also my suggestion above